spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -151,8 +154,13 @@ public String name() {
 
                 @Override
                 public TransactionInfo getTransactionInfo() {
+                    int timeout = globalTrxAnno.timeoutMills();
+                    if (timeout <= 0 || timeout == DEFAULT_GLOBAL_TRANSACTION_TIMEOUT) {
+                        timeout = GLOBAL_TRANSACTION_TIMEOUT;
```

Comment:
if GLOBAL_TRANSACTION_TIMEOUT is  an illegal value，how to process？

Reply:
Handled it in the `static {}` now. Please check it.